llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -598,6 +581,12 @@ def IREELinalgExt_AttentionOp : IREELinalgExt_Op<"attention",
     Value getOutput() {
       return getDpsInitOperand(0)->get();
     }
+    Value getMax() {
```

Comment:
Make this return `std::optional<Value>` since we dont need the value to be specified. So the implementation would return `std::nullopt` if number of results == 1,